### PR TITLE
The "Identify features" shows ellipsoidal and cartesian values

### LINKED ISSUE #2615: [FEATURE] Also show cartesian areas/perimeters in identify results
Original commit: https://github.com/qgis/QGIS/commit/f9be6055375ccf60ec426e561c0f10e88613f63f by nyalldawson

Because users are still getting confused with the difference
between the cartesian areas and ellipsoidal areas, show both
in the identify results dock.

The reasoning here is:

- if a user understands this concept, they will know the correct
one to use, and its good for us to inform them of the difference
here. Plus, it means they immediately see if the ellipsoid
setting is correct and the difference it is making for the
area/length calculation.

- if a user has no idea what the difference is, then we should
make them aware that there's (at least!) two different possible
measurement values. They can then either research what these mean and make
the right choice (and become better informed GIS practitioners),
OR pick a random one - which really is no different then the
previous situation, because an uniformed user is just as likely
to be working in an unsuitable projection with a poor ellipsoid
choice.

In short, we don't try to guess the right choice for users
and instead give them all the information and let them make the
call which value to use.

### PR DESCRIPTION
**Pull request by DelazJ**
Fixes #2615

### CODE REVIEW

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -1296,14 +1296,26 @@ default it will display three kinds of information:
...
+
+    * depending on the geometry type, the cartesian measurements of length,
+      perimeter or area in the layer's CRS units;
+    * depending on the geometry type and if an ellipsoid is set in the project
+      properties dialog for :guilabel:`Measurements`, the ellipsoidal values of
+      length, perimeter or area using the chosen units;
+    * the count of geometry parts in the feature and the number of the part you
+      clicked on;
+    * and the count of vertices in the feature.
+  * coordinates information, uing the project properties :guilabel:`Coordinates
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: uing -> uSing

File: source/docs/user_manual/introduction/general_tools.rst [hunk 2]
```diff
@@ -1296,14 +1296,26 @@ default it will display three kinds of information:
...
+    display` settings:
+
+    * the ``X`` and ``Y`` coordinate values of the clicked point;
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: coordinateS ?

File: source/docs/user_manual/introduction/general_tools.rst [hunk 3]
```diff
@@ -1296,14 +1296,26 @@ default it will display three kinds of information:
...
+    * the number of the closest vertex to the clicked point;
+    * the ``X`` and ``Y`` (and ``Z``/``M`` if applicable) coordinate values of the
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: coordinateS ?

File: source/docs/user_manual/introduction/general_tools.rst (hunk 2 above)
Comment (DelazJ) [reply source/docs/user_manual/introduction/general_tools.rst]: I think "coordinate values" spelling is (the?) right.

**Pull request merged**